syntax = "proto2";

import "Tuple.proto";

package trck;

message Result {
	optional string scalar_a = 1;
	optional string scalar_b = 2;
	repeated string set_x = 3;
	optional fixed64 counter_y = 4;
	repeated trck.Tuple set_z = 5;
}
